src/App.js

```diff
@@ -1,5 +1,11 @@
+import Calculator from './components/Calculator';
+
 function App() {
```

Comment:
- [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md) for this project, " `Use class based components (All components should be class-based).` ", kindly change your `App.js` component which is currently a `functional component` to a `class-based` component.

Reply:
@t-yanick Sure! thank you!